build.gradle

```diff
@@ -29,6 +29,23 @@ allprojects {
     }
 
     version gitVersion()
+
+    if (System.env.CIRCLE_ARTIFACTS) {
+        task copyCiArtifactsToCircle {
```

Comment:
checkout https://github.palantir.build/elements/stemma/blob/develop/circle.yml#L52

We call it in the post test block in circle so that circle runs it after all the things